src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
Yes, I was using it [here](https://github.com/zalmoxisus/social-expert), but those decorators add complexity as well as `react-sortable-hoc`. We need 2 different `ActionList` components - one draggable (with the decorator) and another one not (to turn `draggableActions` prop on / off). Also we need to update local state to drop the element before getting the data from the client part (which takes time in case of the extension). I see your point against using the DOM here, though it does its job. Not sure I'll have time in the next days for it (as other issues are waiting). Maybe someone else could continue the work here.